libstdc++-v3/acinclude.m4

```diff
@@ -1394,6 +1394,14 @@ AC_DEFUN([GLIBCXX_ENABLE_LIBSTDCXX_TIME], [
       uclinux*)
         ac_has_nanosleep=yes
         ac_has_sched_yield=yes
+      zephyr*)
```

Comment:
This will not work since this switch is for `target_os` and `zephyr` is currently not an "OS" (it is a "vendor"). Will need to check for `${target}` here instead.